### ISSUE TITLE
CSoundQT 0.9.7 crash on run

### ISSUE THREAD

**Issue by Neuran** [CsoundQt/CsoundQt#350]
I shoot a video on what happens when I simply run CsoundQT 0.9.7

**Comment by tarmoj** [comment]
Hi!

I don't understand the reason of your problem from this video, using words
to explain might work, too  ;)

There is a couple of things that you can try:
- use later CsoundQt version:
https://github.com/CsoundQt/CsoundQt/releases/tag/v0.9.8.1
- try different audio module (see if it works with null, then portaudio,
then coreaudio)
- try to run the csd in terminal to see if Csound works. I guess Csound is
installed?
- reset your settings in CsoundQt Help ->Reset Preferences

Hope you get it going!
tarmo

Kontakt Ital Rolando (<notifications@github.com>) kirjutas kuupäeval K, 6.
jaanuar 2021 kell 23:27:

**Comment by Neuran** [comment]
Hi tarmoj, 
I've installed 0.9.8.1 and updated Csound to v6.15. (6.13 previously)
After a few initial crashes, it seems to work now (I've used basic instruments). 
I've tried different audio modules as you suggested and "portaudio (callback)" is better than the others. (I'd like to understand why the other modules work so differently in term of audio quality and gap at playback).

Thanks a lot for your help.
